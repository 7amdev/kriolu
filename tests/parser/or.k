mimoria a = falsu ou 1 ou 2;
imprimi a; 
// 1 ou verdadi ou falsu;