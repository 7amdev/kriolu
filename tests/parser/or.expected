<mimoria>
  a
  (falsu ou (1.0 ou 2.0))
<imprimi>
  (get a)